.gitignore

```diff
@@ -2,4 +2,8 @@
 /resources/
 /public/
 node_modules
+_site/
```

Comment:
@justinvp fyi I needed to add these entries. Running `make serve` was generating these.

Reply:
@praneetloke, these should not be needed with Hugo. We used to have similar entries in this file when we were using Jekyll. Are you sure you don’t have older directories left over from before the Hugo migration? Try running `git clean -xdf`.

Reply:
Ohh that might be it. Let me remove them.

Reply:
Confirmed. It was leftover from before. I have reverted this change.